### PR TITLE
Fix grammar in Microservices Security Cheat Sheet

### PR DESCRIPTION
**Pull request by JefSchraag**

### CODE REVIEW

File: cheatsheets/Microservices_Security_Cheat_Sheet.md [hunk 1]
```diff
@@ -6,138 +6,140 @@ The microservice architecture is being increasingly used for designing and imple
...
-    - high latency
-    - should be apply to critical requests
-2. Offline scenario:
-    - to validate incoming token microservice use downloaded service token service public key;
-    - revoked (compromised) token may not be detected
-    - low latency
-    - should be apply to non-critical requests
-In most cases, token-based authentication works over TLS that provides confidentiality and integrity of data in transit.
+Online scenario:
+To validate incoming tokens, the microservice invokes a centralized service token service via network call.
```
Comment (szh) [inline cheatsheets/Microservices_Security_Cheat_Sheet.md:125]: I think these should be formatted as a list.

**Review by szh — CHANGES_REQUESTED** [review]
Overall looks great, just one comment

File: cheatsheets/Microservices_Security_Cheat_Sheet.md [hunk 2]
```diff
@@ -6,138 +6,140 @@ The microservice architecture is being increasingly used for designing and imple
...
-    - it is necessary to build and maintain client library SDKs for every language used in system architecture;
-    - necessity to train every developer on custom authorization service API and integration, and there’s no open source community to source information from.
-4. There is a probability that not all access control policy can be enforced by gateways/proxies and shared authorization library/components, so some specific access control rules still have to be implemented on microservice business code level. In order to do that it is advisiable to have and use by microservice development teams simple questionary/check-list to uncover such security requriments and handle its properly during microservice development.
-5. It is advisable to implement “defense in depth” principle   enforce authorization on:
-    - gateways and proxies level at a coarse level of granularity;
-    - microservice level using shared authorization library/components to enforce fine-granted decisions;
-    - microservice business code level to implement business-specific access control rules.
-6. Access control policy formal procedures like development, approval, rolling-out must be implemented.
```
[the remark this answers is not in the document]
Comment (JefSchraag) [reply cheatsheets/Microservices_Security_Cheat_Sheet.md:71]: That was a mistake. The points are added again.

**Comment by szh** [comment]
Awesome, looks great. There are two lint errors: 
>  33:302  ✓ error  Incorrect usage of the term: “end-user”, use “end user” instead  terminology
  84:315  ✓ error  Incorrect usage of the term: “end-user”, use “end user” instead  terminology

**Review by mackowski — APPROVED** [review]

**Review by szh — APPROVED** [review]
LGTM

**Pull request merged**